build.sbt

```diff
@@ -66,6 +66,7 @@ lazy val root = project
     clientJVM,
     clientJS,
     codegen,
+    codegenSbt,
```

Comment:
Can you also add it to .circleci/config.yml in the test212_jdk8 task?

Reply:
I added it as `codegenSbt/test`, although there are currently no tests. Should it be compile for now instead?

Reply:
That's fine, it will just compile in that case.